client/src/components/ProductDetails.jsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+// import PropTypes from "prop-types";
+
+const ProductDetails = ({ products }) => (
+  <div className="product">
+    {products.map(product => (
+      <div>
+        <div className="brand">Trailblazer</div>
+        <div className="name">{product.name}</div>
+        <div className="rating">
+          {product.rating}
+          <span className="item-num">{product.itemNum}</span>
+        </div>
+        <div>{product.reviewCount}</div>
+        <div>{`$${product.price}.00`}</div>
+        <div>{product.color}</div>
+        <div className="details-image">
+          <img src="" alt="" />
+        </div>
+        <div className="quantity">Quantity</div>
+        <div className="quantity-button" />
+        <input type="submit" value="Add to cart" />
+        <input type="submit" value="Add to wish list" />
+      </div>
+    ))}
+  </div>
+);
+
+// ProductDetails.propTypes = {
```

Comment:
Just curious:  Why did you decide to comment out the propTypes code?  I haven't done anything to set it up yet, but it sounds pretty cool

Reply:
I was having an eslint propTypes error when I went down a rabbit hole last night and tried to refactor my code with react router! But ended up going back to my original code. You're right though, will probably look into later!